.github/workflows/release.yaml

```diff
@@ -16,11 +16,36 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Tag
-        uses: tvdias/github-tagger@v0.0.2
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: "1.14.5"
+
+      - name: Cache
+        uses: actions/cache@v2
         with:
-          repo-token: "${{ secrets.CI_TOKEN }}"
-          tag: "${{ github.event.inputs.version }}"
+          path: ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+
+      - name: Install tools
+        run: ./setup.sh
+
+      - name: Tag
+        run: |
+          git config user.name "$GITHUB_ACTOR"
+          git config user.email "$GITHUB_ACTOR@users.noreply.github.com"
+
+          tag='${{ github.event.inputs.version }}'
+          git tag --annotate --message "Tag for release $tag" "$tag"
+          git push "https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/$GITHUB_REPOSITORY" "refs/tags/$tag"
```

Comment:
afaik the checkout action configures git so you can push back to the same repo w/o having to deal with tokens?

Reply:
will check that, thanks

Reply:
there is however a need to configure author etc: https://github.com/actions/checkout/issues/233 - but I found an action which can do that. give me 2 sec

Reply:
I'm already doing that.

Reply:
https://github.com/fregante/setup-git-token - though marginally helpful.

Reply:
Verify pushing works without specifying the token